### PR TITLE
some fix

### PR DESCRIPTION
**Pull request by deanfirdianza**
In this PR, we have:
- Fix random relation on migration.
- Add eager loading on Pinjaman to Jaminan.
- Fix drop soft delete migration.

### CODE REVIEW

File: database/seeds/JaminanTableSeeder.php [hunk 1]
```diff
@@ -11,6 +11,6 @@ class JaminanTableSeeder extends Seeder
      */
     public function run()
     {
-        factory(Simpin\Infrastructure\Repository\Jaminan\JaminanEloquentRepository::class,10)->create();
+        // factory(Simpin\Infrastructure\Repository\Jaminan\JaminanEloquentRepository::class,10)->create();
```
Comment (akbariskndr) [inline database/seeds/JaminanTableSeeder.php]: I think we just have to delete this seeder because it is unused.

File: test2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+cek
```
Comment (akbariskndr) [inline test2.txt]: delete this file. not used.

Comment (akbariskndr) [reply test2.txt]: same for tst.txt

**Review by akbariskndr — CHANGES_REQUESTED** [review]
Please take a look at my RC.

File: database/seeds/JaminanTableSeeder.php (hunk 1 above)
Comment (deanfirdianza) [reply database/seeds/JaminanTableSeeder.php]: dah cok https://github.com/AxlAdilla/Simpin/pull/1/commits/3673a529627b124e136fa1b13ecf667e35b8d004

File: test2.txt (hunk 2 above)
Comment (deanfirdianza) [reply test2.txt]: dah diapus https://github.com/AxlAdilla/Simpin/pull/1/commits/6ce3d56f2452a5147fe46f4f4c21e5e7a2aa2edc

**Review by akbariskndr — APPROVED** [review]
LGTM

**Pull request merged**